app/services/cleanup-old-data/delete-old-files.js

```diff
@@ -1,22 +1,14 @@
 const Promise = require('bluebird')
-const fs = require('fs')
 const getClaimDocuments = require('../data/get-claim-documents')
-const log = require('../log')
+const { AWSHelper } = require('../aws-helper')
+const aws = new AWSHelper()
 
 module.exports = function (eligibilityId, claimId, reference) {
   return getClaimDocuments('ExtSchema', reference, eligibilityId, claimId)
     .then(function (claimDocuments) {
       claimDocuments.forEach(function (document) {
         if (document.Filepath) {
-          try {
-            fs.unlinkSync(document.Filepath)
-          } catch (error) {
-            if (error.code === 'ENOENT') {
-              log.error(`File for Claim ${claimId} with path ${document.Filepath} does not exist`)
-            } else {
-              throw error
-            }
-          }
+          aws.delete(document.Filepath)
```

Comment:
think this returns a promise, won't this swallow any errors?

Reply:
or rather maybe it should return a promise to allow the same error behaviour as before?

Reply:
I'm currently rethinking my understanding of the Promise stuff anyway so I might just delete my post and ask again later :)